### PR TITLE
Clarifying the behavior of PSA sync

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11+

Issue:
N/A

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/psa-clarifications/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-synchronization_understanding-and-managing-pod-security-admission

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@stlaz Can you please look?

http://file.rdu.redhat.com/~ahoffer/2022/psa-clarifications/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-synchronization_understanding-and-managing-pod-security-admission

File: modules/security-context-constraints-psa-synchronization.adoc [hunk 1]
```diff
@@ -18,18 +18,14 @@ Applying pods directly might use the SCC privileges of the user who runs the pod
...
 
-* All namespaces that are prefixed with `openshift-`, except for `openshift-operators`
 * `default`
 * `kube-node-lease`
 * `kube-system`
 * `kube-public`
 * `openshift`
+* All system-created namespaces that are prefixed with `openshift-`, except for `openshift-operators`
+
+By default, all namespaces that have an `openshift-` prefix are not synchronized. You can enable synchronization for any user-created [x-]`openshift-*` namespaces by setting the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `true`. You cannot enable synchronization for any system-created [x-]`openshift-*` namespaces, except for `openshift-operators`.
```
Comment (bscott-rh) [inline modules/security-context-constraints-psa-synchronization.adoc:30]:
```suggestion
By default, {product-title} will not synchronize namespaces that have an `openshift-` prefix. 
```
Does it make sense to reword this to avoid passive voice?

Comment (stlaz) [reply modules/security-context-constraints-psa-synchronization.adoc:30]: The original passive voice still sounds the best to me here

**Comment by stlaz** [comment]
LGTM

**Comment by bergerhoffer** [comment]
@xingxingxia Can you please QE review this update?

http://file.rdu.redhat.com/~ahoffer/2022/psa-clarifications/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-synchronization_understanding-and-managing-pod-security-admission

**Comment by xingxingxia** [comment]
/lgtm
However, for `"Security context constraint synchronization with pod security standards"` section and `"Controlling pod security admission synchronization"` section, one has an `"Important"` note, the other has a `"NOTE"` note. Though with a bit not exactly same wording, they actually intend to tell the totally same explanation about the different namespaces. I understand they are in different `.adoc` modules, but from user end, the explanation shows twice in the same page `"Understanding and managing pod security admission"`, won't they look duplicate from within one same page?

**Comment by bergerhoffer** [comment]
> /lgtm However, for `"Security context constraint synchronization with pod security standards"` section and `"Controlling pod security admission synchronization"` section, one has an `"Important"` note, the other has a `"NOTE"` note. Though with a bit not exactly same wording, they actually intend to tell the totally same explanation about the different namespaces. I understand they are in different `.adoc` modules, but from user end, the explanation shows twice in the same page `"Understanding and managing pod security admission"`, won't they look duplicate from within one same page?

@xingxingxia Good catch! Not sure how I didn't notice that similar note right below. We do sometimes want to have the important notes in each section, because they are modules and technically could be viewed separately or on another page sometime. But I agree that back-to-back like this looks very duplicative.

I've moved this detailed note down into the actual procedure section: http://file.rdu.redhat.com/~ahoffer/2022/psa-clarifications/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-opting_understanding-and-managing-pod-security-admission

And updated the general info section to have just a basic note saying that system-created namespaces are permanently disabled, but that you can enable sync on other namespaces. (And then they'll just have to go to the other section for the specifics).   http://file.rdu.redhat.com/~ahoffer/2022/psa-clarifications/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-synchronization_understanding-and-managing-pod-security-admission

Let me know what you think of this!

@stlaz If you can also weigh in on whether this approach is okay / whether my condensed note in the above section reads okay

**Comment by stlaz** [comment]
lgtm
